app/lib/adjustments.rb

```diff
@@ -0,0 +1,31 @@
+class Adjustments
+
+  attr_reader :discount_threshold, :discount_rate, :item_discount_rate, :item_on_discount
+
+  def initialize
+    @discount_threshold = 50
+    @discount_rate = 0.95
+    @item_on_discount = "Muffin"
+    @item_discount_rate = 0.90
+  end
+
+  def discount(order_subtotal)
+    order_subtotal > discount_threshold ? 
+    (order_subtotal * discount_rate).round(2) : order_subtotal
+  end
+
+  def item_discount(order)  
+    order.each do |item, array| 
+      (order[item] = [ array.first, (array.last*item_discount_rate).round(2) ]) if item.include?(item_on_discount)
```

Comment:
Line is too long. [114/80]<br>Surrounding space missing for operator '*'.<br>Space inside square brackets detected.